controllers/articles.controller.js

```diff
@@ -13,5 +13,20 @@ exports.getArticle = (req, res, next) => {
     .then((article) => {
         res.status(200).send({article})
     })
-    .catch(next);
+    .catch(next)
+}
+
+exports.getArticleComment = (req, res, next) => {
+    const {article_id} = req.params
+
+    const articlePromises = [selectArticleComment(article_id)]
+    if(article_id){
+        articlePromises.push(checkArticleExists(article_id))
+    }
+    Promise.all(articlePromises)
```

Comment:
If we add the checkArticleExists function to our promise.all then if the article doesn't exist that model will send a promise reject.
Would save having an if statement and pushing to an array.